index.html

```diff
@@ -751,8 +783,8 @@ <h3 id="configuration">Configuration</h3>
       </tr>
       <tr id="input-value">
         <td><b>inputValue</b></td>
-        <td><i>''</i></td>
-        <td>Input field initial value.</td>
+        <td><i>''<br>Promise</i></td>
+        <td>Input field initial value. If the input type is <strong>text</strong>, <strong>email</strong>, <strong>number</strong>, <strong>tel</strong> or <strong>textarea</strong> a Promise is accepted as inputValue.</td>
```

Comment:
> a Promise is accepted as inputValue

a Promise **can be** accepted as inputValue. [See the example](#inputValue-promise).